tests/core/cohere/_utils/test_convert_message_params.py

```diff
@@ -0,0 +1,68 @@
+"""Tests the `cohere._utils.convert_message_params` function."""
+
+import pytest
+from cohere.types import ChatMessage
+
+from mirascope.core.base import AudioPart, BaseMessageParam, ImagePart, TextPart
+from mirascope.core.cohere._utils.convert_message_params import convert_message_params
+
+
+def test_convert_message_params() -> None:
+    """Tests the `convert_message_params` function."""
+
+    message_params: list[BaseMessageParam | ChatMessage] = [
+        ChatMessage(message="Hello"),
+        BaseMessageParam(role="user", content="Hello"),
+        BaseMessageParam(
+            role="user",
+            content=[
+                TextPart(type="text", text="Hello"),
```

Comment:
you can just resolve this comment, but I'm realizing that this path should never be accessible since we only parse into parts if using multimodal. Still worth having this here I think for coverage, but worth noting I think.

Reply:
yeah its pretty much never accessible, but for coverage